src/Packetery/Module/Checkout.php

```diff
@@ -710,7 +710,9 @@ public function getShippingRates(): array {
 
 			$optionId = self::CARRIER_PREFIX . $carrier->getId();
 
-			if ( in_array( $optionId, $disallowedShippingRateIds, true ) ) {
+			// Bugfix for wrong data already stored in database.
+			$optionIdFallback = str_replace( self::CARRIER_PREFIX, 'packetery_carrier_packetery_carrier_', $optionId );
+			if ( in_array( $optionId, $disallowedShippingRateIds, true ) || in_array( $optionIdFallback, $disallowedShippingRateIds, true ) ) {
```

Comment:
$optionIdFallback je hack , který v kódu nechceme.  Prosím dát pryč a problém vyřešit formou migrací.